src/constants/server-error-messages.ts

```diff
@@ -7,9 +7,9 @@ export const SERVER_ERROR_MESSAGES = {
   COMPLETION_ERROR: (action: string) => `An error occurred while ${action}.`,
   CONSOLE_COMPLETION_ERROR: (action: string) => `Error ${action}.`,
   DELETE_NOT_ALLOWED: (entity: string) =>
-    `You are not allowed to delete this ${entity}.`,
-  DUPLICATE_ENTRY: (field: string, entity: string, context: string) =>
-    `A ${entity} with the same ${field} already exists in the ${context}.`,
+    `You cannot delete this ${entity} at least one ${entity} must remain.`,
```

Comment:
It would probably more makes sense if you could add a semicolon like below.
```suggestion
    `You cannot delete this ${entity}; at least one ${entity} must remain.`,
```

Reply:
Thank you for your review! I fixed it!